src/fonts/v5icons.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg">
+<metadata>Generated by Fontastic.me</metadata>
+<defs>
+<font id="branches" horiz-adv-x="512">
+<font-face font-family="branches" units-per-em="512" ascent="480" descent="-32"/>
+<missing-glyph horiz-adv-x="512" />
+
+<glyph glyph-name="branching-off" unicode="&#97;" d="M399 263l-281 0 0 190 281 0z m-246 35l211 0 0 120-211 0z m282-240l-142 0 0 123 142 0z m-107 35l72 0 0 53-72 0z m-87 187l35 0 0-74-35 0z m-23-222l-142 0 0 123 142 0z m-107 35l72 0 0 53-72 0z m50 138l194 0 0-35-194 0z m-31-2l35 0 0-61-35 0z m222 0l35 0 0-61-35 0z"/>
```

Comment:
@bzabos , I'm wondering if we still need these icons ? We're now using new icons from designers, right ?